### PR TITLE
Harden gpicview.profile

### PR DESCRIPTION
**Pull request by glitsj16**

### CODE REVIEW

File: etc/gpicview.profile [hunk 1]
```diff
@@ -14,9 +14,10 @@ include disable-interpreters.inc
 include disable-passwdmgr.inc
 include disable-programs.inc
 
-include whitelist-var-common.inc
```
Comment (SkewedZeppelin) [inline etc/gpicview.profile]: removed?

Comment (glitsj16) [reply etc/gpicview.profile]: Yes I tested quite extensively without including whitelist-var-common.inc on both Arch and Ubuntu. Gpicview works just fine without it (as does eog). Do you want me to put it back in?

Comment (SkewedZeppelin) [reply etc/gpicview.profile]: yes please
there are lots of files under /var that programs shouldn't have access to

Comment (glitsj16) [reply etc/gpicview.profile]: Fixed in https://github.com/netblue30/firejail/pull/2476.

**Pull request merged**